propagators/opentelemetry-propagator-grpc-census-binary/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@opentelemetry/propagator-grpc-census-binary",
+  "version": "0.7.0",
```

Comment:
Due to https://github.com/open-telemetry/opentelemetry-js-contrib/pull/40, we should use `0.8.0` here.